test/integration/server.h

```diff
@@ -207,6 +207,13 @@ class IntegrationTestServer : Logger::Loggable<Logger::Id::testing>,
   }
   void start(const Network::Address::IpVersion version);
   void start();
+
+  void waitForCounterGe(const std::string& name, uint64_t value) {
```

Comment:
I know there are other cases where we are doing this (In fact IIRC you implemented one at some point, and I think there may be others)? I might be making this up. If there are can we replace with this?